simply-rets-api-helper.php

```diff
@@ -1373,6 +1394,10 @@ public static function srResidentialResultsGenerator( $response, $settings ) {
                     $compliance_markup
                   </div>
                 </div>
+                <div class="sr-screen-reader" itemprop="geo" itemscope itemtype="http://schema.org/GeoCoordinates">
+                  <meta content="$lat" itemprop="latitude">
+                  <meta content="$lng" itemprop="longitude"
```

Comment:
@wesleycole - sorry, what I mean was that it's missing a closing `>` character:

```
                  <meta content="$lat" itemprop="latitude">
                  <meta content="$lng" itemprop="longitude"
```

That's the only thing I saw, though.  If you want to push a fix for that, I can merge it in. I can also push a fix for that if you're busy :)

Other than that, I'm down! Great addition, thank you!